src/pages/MemberProfilePage.tsx

```diff
@@ -169,7 +169,7 @@ const MemberProfilePage: React.FC = () => {
   const { logout, member: authMember } = useAuth();
   const navigate = useNavigate();
   const [isEditing, setIsEditing] = useState(false);
-  const [member, setMember] = useState<Member | null>(authMember || null);
+  const [member, setMember] = useState<Member | undefined>(authMember);
```

Comment:
You shouldn't need `| undefined`

Reply:
I think it might if you pass an optional in, i.e. `useState<Member>(authMember)`. can switch it though to useEffect